src/Functions/GatherUtils/Sources.h

```diff
@@ -122,9 +122,15 @@ struct NumericArraySource : public ArraySourceImpl<NumericArraySource<T>>
     }
 };
 
+
+/// The method can be virtual or not depending on the template parameter.
```

Comment:
What method? And what is an original issue with overrides?

Reply:
The methods of ISource.

It isn't possible to specify `override` if the method is not virtual in the base class.

Reply:
Now I see.